substrate/network/src/import_queue.rs

```diff
@@ -249,6 +253,16 @@ fn import_many_blocks<'a, B: BlockT>(
 	let count = blocks.len();
 	let mut imported = 0;
 
+	trace!(target:"sync", "Starting import of {} blocks{}", count,
+		match (
```

Comment:
Let's avoid such big expressions as an argument, especially when used in positional formatting.